packages/config/src/dynamo/__tests__/config.imagery.test.ts

```diff
@@ -1,8 +1,9 @@
 import { Epsg, EpsgCode, NamedBounds, QuadKey, TileMatrixSet, TileMatrixSets } from '@basemaps/geo';
 import o from 'ospec';
 import { createSandbox } from 'sinon';
+import { Config } from '../..';
```

Comment:
My first time see this import? `'../..'`

Reply:
`..` resolves to `../index.ts` so `../..` is `../../index.ts`